src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -522,43 +413,41 @@ export default Vue.extend({
         displayOptions.arrowColor = inactiveGray;
       }
     },
-    closeCollegeDropdownIfOpen(event, i) {
+    closeCollegeDropdownIfOpen(_: unknown, i: number) {
       this.closeDropdownIfOpen('college', i);
     },
-    closeMajorDropdownIfOpen(event, i) {
+    closeMajorDropdownIfOpen(_: unknown, i: number) {
       this.closeDropdownIfOpen('major', i);
     },
-    closeMinorDropdownIfOpen(event, i) {
+    closeMinorDropdownIfOpen(_: unknown, i: number) {
       this.closeDropdownIfOpen('minor', i);
     },
```

Comment:
I feel like these functions are repetitive and could be refactored into one function. Could this be handled when factoring out a dropdown component?

Reply:
Similarly for the functions `selectCollege()`, `selectMajor()`, `selectMinor()` and `removeCollege()`, etc.

Reply:
Yes all of them will be handled